README.md

```diff
@@ -1 +1,260 @@
-# group-project-space-travelers-hub
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img/>
+  <br/>
+
+  <h3><b>Space Traveles' Hub</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+
+# 📖 Space Traveles' Hub <a name="about-project"></a>
+
+It is a React Page that has the ability to show the mission and rockets from API
+and has the ability to reserve rocket and join a mission and added it to the profile
+and has many pages by React Routes
+
+
+## 🛠 Built With <a name="built-with"></a>
+-HTML
+-CSS
+-JS
+-React
+-Redux
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Single page application
+- Has the ability to Change between the pages by the navbar
+- Has the ability to join and reserve and show it in the profile
+- Has the ability to fetch from the API
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Comming soon
+[Live view]()
```

Comment:
- [ ] @Team your live demo link is not working, you can choose to delete the entire section or add a working link of your live demo